src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -178,6 +178,12 @@ public void Configure(OpenIddictServerOptions options)
         options.Scopes.Add(Scopes.Phone);
         options.Scopes.Add(Scopes.Profile);
         options.Scopes.Add(Scopes.Roles);
+
+        // Note: caching is enabled for both authorization and end session requests to allow sending
+        // large POST authorization and end session requests, but can be programmatically disabled, as the
+        // authorization and end session views support flowing the entire payload and not just the request_uri.
+        options.EnableAuthorizationRequestCaching = true;
```

Comment:
Note: these options were moved to `OpenIddictServerOptions` (instead of `OpenIddictServerAspNetCoreOptions`) as part of the OAuth 2.0 Pushed Authorization Requests introduction. The old properties are still there but obsolete and no-op.